pkgs/http_profile/README.md

```diff
@@ -1,2 +1,68 @@
-An **experimental** package that allows HTTP clients outside of the Dart SDK
-to integrate with the DevTools Network tab.
+[![pub package](https://img.shields.io/pub/v/http_profile.svg)](https://pub.dev/packages/http_profile)
+[![package publisher](https://img.shields.io/pub/publisher/http_profile.svg)](https://pub.dev/packages/http_profile/publisher)
+
+A package that allows HTTP clients outside of the Dart SDK to integrate with
+the DevTools Network tab.
+
+## Using
+
+`HttpClientRequestProfile.profile` returns an `HttpClientRequestProfile` object.
+Populating the fields of that object with information about an HTTP request and
+about the response to that request will make that information show up in the
+DevTools Network tab.
+
+```dart
+import 'package:http_profile/http_profile.dart';
+
+void main() {
+  HttpClientRequestProfile.profilingEnabled = true;
```

Comment:
I'd remove this because it is not idiomatic for real code.

Reply:
Done